documentation/src/main/asciidoc/topics/ref_protostream_types.adoc

```diff
@@ -1,10 +1,6 @@
 [id='protostream_types-{context}']
-= Supported Types
-{brandname} uses a ProtoStream API to encode and decode Java objects into
-Protocol Buffers (Protobuf); a language-neutral, backwards compatible format.
-
-ProtoStream can handle the following types for keys and values, as well as the
-unboxed equivalents in the case of primitive types:
+= ProtoStream types
+{brandname} uses a ProtoStream API that can handle the following types for keys and values, as well as the unboxed equivalents in the case of primitive types:
```

Comment:
```suggestion
{brandname} uses the ProtoStream library, which can natively handle the following types for keys and values, as well as the unboxed equivalents in the case of primitive types:
```

I think I've noticed one more place where the text says "ProtoStream API" but it's actually talking about the implementation, not the API.